1.adoc

```diff
@@ -23,6 +23,12 @@ Platforma:: Webový prohlížeč (editor je komponenta použitelná v rámci web
 == Product statement
 == Needs assesment
 == Business requirements
+
+Cílem editoru je efektivní použití pro danou cílovou skupinu.
+Prioritou při editování dokumentu je jeho obsah, nikoli vzhled.
+Přesto že se jedná o editor dokumentů, je snahou přiblížit se kodérským editorům, kde má uživatel přímou kontrolu nad zdrojovým kódem dokumentu, než nabízet omezený WYSIWYG editor.
```

Comment:
"Zároveň ale neobětováváme přínosy WYSIWYGu, neboť poskytujeme real-time náhled."

Reply:
Navrhuju trpnej rod a vyhnout se slovu prinosy, protoze to zni, jako ze realtime nahled resi vsechno co je dobryho na wysiwyg.

"Pohodlnost WYSIWYG editoru je kompenzována real-time náhledem."

Reply:
fixed 0c501aa10d3902aa7bb87a1a65b65611071e4bf1